flask_caching/backends/filesystem.py

```diff
@@ -184,7 +183,7 @@ def set(self, key, value, timeout=None, mgmt_element=False):
             )
             with os.fdopen(fd, "wb") as f:
                 pickle.dump(timeout, f, 1)
-                pickle.dump(value, f, pickle.HIGHEST_PROTOCOL)
+                pickle.dump(value, f, 4)  # 4 was the highest protocol
```

Comment:
`HIGHEST_PROTOCOL` still exists in Python 3: https://docs.python.org/3/library/pickle.html#pickle.HIGHEST_PROTOCOL